carapace-server/src/main/java/org/carapaceproxy/server/mapper/StandardEndpointMapper.java

```diff
@@ -352,11 +331,11 @@ public MapResult map(HttpRequest request, String userId, String sessionId, Backe
             if (!route.isEnabled()) {
                 continue;
             }
-            RoutingKey matchResult = route.matches(request);
```

Comment:
Do not drop this abstraction, in the future it will carry more information about the decided route/user